app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -243,6 +243,11 @@ def parse_pod(pod)
         end
       end
 
+      conditions = pod.status.conditions
+      new_result[:container_conditions] = conditions.to_a.collect do |condition|
+        parse_pod_condition(condition)
+      end
```

Comment:
Please unify this code with the above one (the one of the nodes).

You can use just a single method I believe `parse_pod_conditions` (plural) where you do the cycle and you return the hashes.